pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_es.html

```diff
@@ -85,7 +85,7 @@
     <td style="padding: 50px 0 0 0;">
       <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Estimado(a) {{proxy_firstName}},</p>
 
-      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Gracias por unirse al Proyecto sobre Osteosarcoma y contarnos sobre la experiencia de su hijo(a) con el cáncer. Estamos sumamente agradecidos de que haya inscrito a su hijo(a) en el estudio. A medida que crezca, nuestros requisitos para que continúe participando en el estudio cambiarán y necesitaremos comunicarnos directamente con él o ella para que proporcione su consentimiento ininterrumpido. En unos pocos meses, su hijo(a) alcanzará la edad en la que tendrá que firmar un formulario de consentimiento para seguir participando en el estudio.</p>
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Gracias por unirse al Proyecto sobre Osteosarcoma y contarnos sobre la experiencia de su hijo(a) con el cáncer. Estamos sumamente agradecidos de que haya inscrito a su hijo(a) en el estudio. A medida que crezca, nuestros requisitos para que continúe participando en el estudio cambiarán.</p>
```

Comment:
based on source - might need to add the following after "..el estudio cambiarán..":

> El proceso para mantenerlos
en el estudio es comunicarnos directamente con los participantes una vez que se conviertan en
adultos para prestar su consentimiento explícito.
En unos meses, su hijo(a) alcanzará la edad en la que necesitará renovar su consentimiento para
continuar participando en el estudio.